CRM/Contribute/Form/Contribution/Confirm.php

```diff
@@ -740,13 +740,13 @@ public function postProcess() {
    *
    * Comments from previous refactor indicate doubt as to what was going on.
    *
-   * @param int $contributionTypeId
+   * @param int $financialTypeId
    *
    * @return null|string
    */
-  public function wrangleFinancialTypeID($contributionTypeId) {
-    if (isset($paymentParams['financial_type'])) {
-      $contributionTypeId = $paymentParams['financial_type'];
```

Comment:
This is a bit strange as ```$paymentParams``` is not a parameter nor initialised inside the function yet this is in existence, seems like something to do with past refactoring. Also we should change the variable name to ```financialTypeId```  from ```contributionTypeId``` as what we all know and in past why that column was renamed :) 

so chnages here would be:
```php
public function wrangleFinancialTypeID($financialTypeId) {
 if (empty($financialTypeID) && !empty($this->_values['pledge_id'])) {
   $contributionTypeId = CRM_Core_DAO::getFieldValue('CRM_Pledge_DAO_Pledge',
         $this->_values['pledge_id'],
         'financial_type_id'
       );
}
 return $financialTypeId;
}
```

Reply:
@monishdeb Thanks for the review.  I've updated that function to match what you suggested as that's much cleaner.